src/platform/backends/qemu/qemu_virtual_machine.cpp

```diff
@@ -343,9 +343,24 @@ void mp::QemuVirtualMachine::start()
     vm_process->write(qmp_execute_json("qmp_capabilities"));
 }
 
-void mp::QemuVirtualMachine::shutdown()
+void mp::QemuVirtualMachine::shutdown(const bool force)
 {
-    if (state == State::suspended)
+    if (force)
+    {
+        state = State::off;
+        mpl::log(mpl::Level::info, vm_name, "Forced shutdown");
```

Comment:
Could this say instead

```suggestion
        mpl::log(mpl::Level::info, vm_name, "Forcing shutdown");
```

Just to be clear that the shutdown did not occur just yet. I think it would make more sense given the order of the logs, and just in case the daemon crashes when attempting to kill (or if there is an apparmor denial or something).